src/commands/testing/spec/testing.spec.ts

```diff
@@ -769,6 +770,62 @@ testteardown,tests/testteardown.sas,sasjs_test_id,not provided,,${testUrlLink(
       expect(coverageLcov).toEqual(expectedCoverageLcov)
     })
   })
+
+  describe('General', () => {
+    const target: Target = generateTarget(ServerType.SasViya)
+
+    const testUrl = (test: string) =>
+      `${getTestUrl(
+        target,
+        `/Public/app/cli-tests/${target.name}/tests/${test}`
+      )}&_contextName=${encodeURIComponent(target.contextName)}`
+    const testUrlLink = (test: string) => `"=HYPERLINK(""${testUrl(test)}"")"`
+
+    beforeAll(async () => {
+      await createTestApp(__dirname, target.name)
+      await copyTestFiles(target.name)
+      await build(target)
+
+      process.logger = new Logger(LogLevel.Off)
+    })
+
+    beforeEach(() => {
+      setupMocksForSASVIYA()
+    })
+
+    afterAll(async () => {
+      await removeTestApp(__dirname, target.name)
+    })
+
+    it('should fail on tests failing', async () => {
+      const error = await runTest(
+        target,
+        undefined,
+        undefined,
+        undefined
+      ).catch((err: any) => {
+        return err
+      })
+
+      expect(error).toEqual(
+        '5 tests failed to complete!\n 5 tests completed with failures!'
```

Comment:
How do you know that exactly 5 tests should fail?

Reply:
Computation was wrong. I corrected it.

![image](https://user-images.githubusercontent.com/18329105/159277435-57a8bd5e-73b8-4d68-a211-16cd2e63b494.png)